### ISSUE TITLE
About pretrained model settings

### ISSUE THREAD

**Issue by gogyzzz** [taylorlu/ghostvlad-speaker#3]
Hi taylorlu, I want to know the settings of the pretrained model. (epoch, batch size, learning rate decay...)

I used the model and it seems works nicely.

**Comment by taylorlu** [comment]
How did you train it, the model was simply converted from the keras model which shared by https://www.dropbox.com/sh/n96ekf7ilsvkjdp/AACXKDesS2ju5rp6Cyxh2PCva?dl=0

**Comment by gogyzzz** [comment]
I used the default setting from your code. I changed only the number of speakers.
```python
        "sample_rate": 16000,
        "min_duration": 1000,
        "max_duration": 3000,
        "batch_size": 256,
        "epochs": 1000,
        "learning_rate": 0.001,
        "max_grad_norm": 50,
        "decay_steps": 5000,
        "decay_rate": 0.95,
        "vlad_clusters": 8,
        "ghost_clusters": 2,
        "embedding_dim": 512,
        "num_class": 9247
```
I'm training two projects, on (vox_celeb2 dataset) and (my_dataset+vox_celeb1,2).

On only vox_celeb2 training case, I got the lastest training loss. 

Epoch 32, iter 138464: Cost= 20.58, lr= 1.00e-03

How did your cases do?

And in your dropbox link, there are accuracies. But I cannot find them in your code.

Thank you!

**Comment by taylorlu** [comment]
Hi, the **original paper** used `amsoftmax/softmax` as their loss function, but my code was` arcface` loss, the training process was a little tricky too, you need to use a weight λ to control the loss which behave like `softmax` at the beginning and `amsoftmax/arcface` later, since the loss function was hard to train directly.
Strongly recommend you read the papers:
arcface: https://arxiv.org/abs/1801.07698
amsoftmax: https://arxiv.org/abs/1801.05599
sphereface: https://arxiv.org/abs/1704.08063
By the way, you can use softmax loss function to look the performance at first.

**Comment by gogyzzz** [comment]
Thank you!
